modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc

```diff
@@ -1,59 +1,91 @@
-= Upgrade from Community to Subscription
-:description: Upgrading means moving from a Bonita Community edition environment to any Bonita Subscription
+= Upgrade from Community to Enterprise
```

Comment:
Rename page to (validate we can use slash):
Bonita edition change - Community to/from Enterprise
```suggestion
= Bonita edition change - Community to/from Enterprise
```

Reply:
done